primitives/message-lane/src/source_chain.rs

```diff
@@ -16,14 +16,26 @@
 
 //! Primitives of message lane module, that are used on the source chain.
 
-use crate::{InboundLaneData, LaneId};
+use crate::{InboundLaneData, LaneId, MessageNonce};
 
-use frame_support::Parameter;
-use sp_std::fmt::Debug;
+use frame_support::{Parameter, RuntimeDebug};
+use sp_std::{collections::btree_map::BTreeMap, fmt::Debug};
 
 /// The sender of the message on the source chain.
 pub type Sender<AccountId> = frame_system::RawOrigin<AccountId>;
 
+/// Relayers rewards, grouped by relayer account id.
+pub type RelayersRewards<AccountId, Balance> = BTreeMap<AccountId, RelayerRewards<Balance>>;
+
+/// Single relayer rewards.
```

Comment:
```suggestion
/// Single relayer reward.
```

Reply:
I just understood that you meant "Rewards of a single relayer" not "Relayer's single rewards". It is confusing me, but given you seem to be actively opposing changing this I'd rather merge as-is.